blog/_posts/2020-07-15-Tetiana-Dadakova.md

```diff
@@ -0,0 +1,232 @@
+---
+layout: post
+title: "Future blogpost about MABE"
+date: 2020-07-15
+author: Tetiana
+---
+
+# Genome change logging using range map  
+
```

Comment:
I would include a brief intro/overview as to what your project was for this summer (and any goals you had) before diving straight into the details